packages/components/psammead-caption/src/index.jsx

```diff
@@ -11,24 +11,15 @@ import {
   GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
-import {
-  getLongPrimer,
-  GEL_FF_REITH_SANS,
-} from '@bbc/gel-foundations/typography';
+import { getLongPrimer } from '@bbc/gel-foundations/typography';
 import { C_METAL } from '@bbc/psammead-styles/colours';
 import { scriptPropType } from '@bbc/gel-foundations/prop-types';
-
-// Defined separately since in future will need to apply
-// only when the script supports italic text
-const FS_ITALIC = css`
-  font-style: italic;
-`;
+import { getSansItalic } from '@bbc/psammead-styles/font-styles';
 
 const Caption = styled.figcaption`
   ${props => (props.script ? getLongPrimer(props.script) : '')};
   color: ${C_METAL};
-  font-family: ${GEL_FF_REITH_SANS};
-  ${FS_ITALIC};
+  ${({ service }) => (service ? getSansItalic(service) : '')}
```

Comment:
Can you do ```${({ service }) => getSansItalic(service)}```? Bit cleaner, I think `getSansItalic` returns null which should be fine, AFAIK you can return null inside a styled-components inner func

Reply:
On it. Making the change to all the other `components`

Reply:
Done :-)